src/task3.java

```diff
@@ -0,0 +1,18 @@
+public class task3 {
+
+    class Person {
```

Comment:
please declare Person class as a static inner class or moving it outside the task3 class
this is because non-static inner classes hold an implicit reference to their outer class instances, which isn't necessary in this context